dotcom-rendering/src/web/lib/renderElement.tsx

```diff
@@ -580,7 +588,12 @@ export const renderElement = ({
 				/>
 			);
 		case 'model.dotcomrendering.pageElements.SubheadingBlockElement':
-			return <SubheadingBlockComponent key={index} html={element.html} />;
+			return (
+				<SubheadingBlockComponent
+					key={index}
+					html={sanitise(element.html, sanitiserOptions)}
```

Comment:
Question for my own understanding! So in the before example in the PR description

![image](https://user-images.githubusercontent.com/19683595/223202022-131aa813-758c-41ca-a5c0-a0dea3df5ecc.png)

[`SubheadingBlockComponent` is calling `unwrapHtml`](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/web/components/SubheadingBlockComponent.tsx) but this method [cannot find a `matchingFix`](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/model/unwrapHtml.ts#L37) because 

```<h2 id="arcadia-peloponnese> !== <h2>```. Is that correct? And could you please explain the choice of `sanitiserOptions`?